Gemfile.lock

```diff
@@ -207,6 +210,18 @@ GEM
     json (2.3.1)
     jsoneditor-rails (1.0.3)
       sass-rails
+    kaminari (1.2.1)
```

Comment:
good choice. would have thought of the same.
have you met other solutions on the way?

Reply:
[pagy](https://github.com/ddnexus/pagy) seems even better. I came across it afterwards.
Maybe we should consider migrating

Reply:
[pagy](https://github.com/ddnexus/pagy) looks really good